modules/agent-install-mirroring-for-disconnected-registry.adoc

```diff
@@ -0,0 +1,253 @@
+repository, and for `<local_registry_host_port>`, specify the port that it
+serves content on.
+
+.. Export the local repository name:
++
+[source,terminal]
+----
+$ LOCAL_REPOSITORY='<local_repository_name>'
+----
++
+For `<local_repository_name>`, specify the name of the repository to create in your
+registry, such as `ocp4/openshift4`.
+
+.. Export the name of the repository to mirror:
++
+[source,terminal]
+----
+$ PRODUCT_REPO='openshift-release-dev'
+----
++
+For a production release, you must specify `openshift-release-dev`.
++
+[source,terminal]
+----
+$ PRODUCT_REPO='openshift'
+----
+
+
+.. Export the path to your registry pull secret:
++
+[source,terminal]
+----
+$ LOCAL_SECRET_JSON='<path_to_pull_secret>'
+----
++
+For `<path_to_pull_secret>`, specify the absolute path to the file name of the pull secret for your mirror registry that you created.
+
+.. Export the release mirror:
++
+[source,terminal]
+----
+$ RELEASE_NAME="ocp-release"
+----
++
+For a production release, you must specify `ocp-release`.
++
+[source,terminal]
+----
+$ RELEASE_NAME="okd"
+----
+
+.. Export the type of architecture for your server, such as `x86_64`:
++
+[source,terminal]
+----
+$ ARCHITECTURE=<server_architecture>
+----
+
+.. Export the path to the directory to host the mirrored images:
++
+[source,terminal]
+----
+$ REMOVABLE_MEDIA_PATH=<path> <1>
+----
+<1> Specify the full path, including the initial forward slash (/) character.
+
+. Because your mirror host does not have internet access, perform the following actions:
+.. Connect the removable media to a system that is connected to the internet.
+.. Review the images and configuration manifests to mirror, by running the following command:
++
+[source,terminal]
+----
+$ oc adm release mirror -a ${LOCAL_SECRET_JSON}  \
+     --from=quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE} \
+     --to=${LOCAL_REGISTRY}/${LOCAL_REPOSITORY} \
+     --to-release-image=${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}:${OCP_RELEASE} --dry-run
+----
+. In the `install-config.yaml` file, set the following parameters:
+
++
+[source,yaml]
+----
+  pullSecret: "<pull_secret>" <1>
+  sshKey: 'ssh-rsa <ssh_pub_key>'
+  imageContentSources: <2>
+  - mirrors:
+    - my-registry:443/ocp4-release/openshift
+    source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+  - mirrors:
+    - my-registry:443/ocp4-release/openshift
+    source: quay.io/openshift-release-dev/ocp-release
+  additionalTrustBundle: | <3>
+      -----BEGIN CERTIFICATE-----
+      <MY_TRUSTED_CA_CERT>
+      -----END CERTIFICATE-----
+----
+<1> A pull secret for the mirror registry.
+<2> Mirrors for sources and repositories for the release-image content. The certificate file can be an existing, trusted certificate authority,
+or the self-signed certificate that you generated for the mirror registry.
+<3> The certificate file used for the mirror registry.
+
+. Create the `agent-config.yaml` file:
```

Comment:
I don't think its necessary to show the agent-config.yaml file as its not needed for mirroring.

Reply:
Removed this step